mne_bids/write.py

```diff
@@ -1535,6 +1536,9 @@ def write_raw_bids(
     extra_columns_descriptions : dict | None
         A dictionary that maps column names of the ``event_metadata`` to descriptions.
         Each column of ``event_metadata`` must have a corresponding entry in this.
+    electrodes_tsv_task : bool
+        Add the ``task-`` entity to the ``BIDSPath`` of the
+        ``electrodes.tsv`` file. Defaults to ``False``.
```

Comment:
```suggestion
        Add the ``task-`` entity to the ``electrodes.tsv`` filename. Defaults to ``False``.
```

I think that this makes more sense but let me know if you disagree or if I am misunderstanding something.